apps/omg/lib/omg/state/core.ex

```diff
@@ -405,7 +405,6 @@ defmodule OMG.State.Core do
   def exit_utxos([%{tx_hash: _} | _] = piggybacks, %Core{utxos: utxos} = state) do
     piggybacks
     |> Enum.map(fn %{tx_hash: tx_hash, output_index: oindex} ->
-      # oindex in contract is 0-7 where 4-7 are outputs
```

Comment:
I think this comment might have been helpful. This subtraction is otherwise rather cryptic